Sources/SimultaneouslyScrollView/DefaultSimultaneouslyScrollViewHandler.swift

```diff
@@ -2,8 +2,18 @@
 import Combine
 import UIKit
 
+internal class ScrollViewDecorator {
```

Comment:
Optional: I personally prefer to put every class into a separate file.

Reply:
[1f4a55e](https://github.com/stonko1994/SimultaneouslyScrollView/pull/14/commits/1f4a55ea387afe19785c98fc80402f171503482d)